hub/api/tests/test_api.py

```diff
@@ -560,6 +560,12 @@ def test_htype(memory_ds: Dataset):
     segment_mask = memory_ds.create_tensor("segment_mask", htype="segment_mask")
     keypoints_coco = memory_ds.create_tensor("keypoints_coco", htype="keypoints_coco")
     point = memory_ds.create_tensor("point", htype="point")
+    point_cloud = memory_ds.create_tensor(
+        "point_cloud", htype="point_cloud", sample_compression="las"
+    )
+    memory_ds.create_tensor(
+        "point_cloud_calibration_matrix", htype="point_cloud.calibration_matrix"
```

Comment:
can you point me to some documentation around this? I'm confused about this

Reply:
I haven't included in documentation point_cloud.calibration_matrix, what is confusing about this htype?

Reply:
why is it used? should the user always create it? did we finalize it's name?

Reply:
No, it is not mandatory, but it is used to map 3d objects on 2d image

Reply:
Regarding name, I proposed the name to Ivo, and he was not against it